regex-assembly/932175.ra

```diff
@@ -14,10 +14,12 @@
 
   ##! match white space between command and arguments
   \b\s+
+  \b\s+[+\-][a-z]+\s+
+  \b\s+[+\-][a-z]+\+\s+
```

Comment:
These two lines have nothing to do with white space. They should be moved after the `##!=>` (without the `\b\s+` prefix):

```
  ##! match white space between command and arguments
  \b\s+
  ##!=>

  ##! match flags
  (?:[+\-][a-z]+\s+)?
  ##!=>
```